gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/CheckComposeCompilerPlugin.kt

```diff
@@ -28,20 +28,20 @@ internal fun Project.checkComposeCompilerPlugin() {
     }
 }
 
+internal const val minimalSupportedKgpVersion = "2.1.0"
 internal const val minimalSupportedKgpVersionError = "e: Configuration problem: " +
-        "Minimal supported Kotlin Gradle Plugin version is 2.0.0"
-internal const val newCompilerIsAvailableVersion = "2.0.0"
+        "Minimal supported Kotlin Gradle Plugin version is $minimalSupportedKgpVersion"
 internal const val newComposeCompilerKotlinSupportPluginId = "org.jetbrains.kotlin.plugin.compose"
 internal const val newComposeCompilerError =
-    "Since Kotlin 2.0.0 to use Compose Multiplatform " +
-            "you must apply \"$newComposeCompilerKotlinSupportPluginId\" plugin." +
+    "Starting with Compose Multiplatform 1.6.10, " +
```

Comment:
It depends on Kotlin + compiler plugin, not on Compose Multiplatform version

Reply:
Both statements are valid, aligning the error message with documentation is a good idea.